include/libdnf/conf/const.hpp

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright (C) 2020 Red Hat, Inc.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef LIBDNF_CONST_CONFIG_CONST_HPP
+#define LIBDNF_CONST_CONFIG_CONST_HPP
+
+#include <string>
+#include <vector>
+
+namespace libdnf {
+
+constexpr const char * PERSISTDIR = "/var/lib/dnf";
+constexpr const char * SYSTEM_CACHEDIR = "/var/cache/dnf";
+
+constexpr const char * CONF_FILENAME = "/etc/dnf/dnf.conf";
```

Comment:
Where is the `/etc/dnf/conf.d` setup?

Reply:
Thanks for comments. I wrote notes. The code is under development. I'm trying to reuse old code and adopt it to libdnf5.

Reply:
Why '/etc/dnf/conf.d' ?

Reply:
That's what's in the `dnf` PR: https://github.com/rpm-software-management/dnf/pull/1414